### PR TITLE
Add support for configurable binding address

### PR DESCRIPTION
**Pull request by aaccioly**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -52,15 +54,14 @@ type AwsConfig struct {
...
-	godotenv.Load(".env")
-	if os.Getenv("DB_ENGINE") == "" {
-		os.Setenv("DB_ENGINE", "lmdb")
-	}
+	_ = godotenv.Load(".env")
 
 	return Config{
 		OwnerNpub:                        getEnv("OWNER_NPUB"),
-		DBEngine:                         getEnv("DB_ENGINE"),
+		DBEngine:                         getEnvString("DB_ENGINE", "lmdb"),
```
Comment (barrydeen) [inline config.go:61]: Why was this changed? It should default to badger

File: main.go [hunk 2]
```diff
@@ -48,8 +48,10 @@ func main() {
 
 	http.HandleFunc("/", dynamicRelayHandler)
 
-	log.Printf("🔗 listening at http://localhost:3355")
-	http.ListenAndServe("0.0.0.0:3355", nil)
+	addr := fmt.Sprintf("%s:%d", config.RelayBindAddress, config.RelayPort)
```
Comment (barrydeen) [inline main.go:51]: Good change

**Review by barrydeen — CHANGES_REQUESTED** [review]

File: config.go (hunk 1 above)
Comment (aaccioly) [reply config.go:61]: It was already defaulting to LMDB when no env parameter was set. Look at line 57 above. I assumed you did this intentionally - wanting new installs to default to Badger, but not to touch old installs already using LMDB without specifying the env variable. If you'd prefer to switch even old installs to use Bager by default, feel free to change the string, or let me know and I can make the change myself.

Comment (barrydeen) [reply config.go:61]: Ah yes you're right lol, thanks for knowing my relay better than me. Too many relays! Thanks for the pr
